### DISCUSSION TITLE
CO2 emissions from unknown sources in Italy

### DISCUSSION THREAD

**Discussion by Raffaelemzz** [electricitymaps/electricitymaps-contrib#6344] [General]
It seems that Enel has been authorised to permanently close the coal-fired units of the 'Andrea Palladio' power plant in Fusina from **31 December 2023**.
Therefore, coal power capacity in Northern Italy will be reduced to 336 MW (Monfalcone).
https://www.staffettaonline.com/articolo.aspx?id=381015

This introduces the issue of CO2 emissions from unknown sources.
Production from unknown sources in Northern Italy is often around 1.5 - 2.5 GW. The total capacity from coal and oil is 336 MW and 42 MW respectively.
So most of the power generated from unknown sources certainly does not come from oil or coal. The average emission of 700 gCO2/kWh seems exaggerated to me.

This argument regarding the area of Central Northern Italy is even simpler.
The capacity is zero for coal and 16 MW for oil.

**Comment by VIKTORVAV99** [comment]
I'll see what we can do about this but the most accurate way to fix this would be to reach out to TERNA and ENTSO-E and see if they can assign the other values to the correct production mode.
